resources/lang/vi.json

```
@@ -136,6 +136,9 @@
     "discover_now" : "Xem ngay",
     "discount" : "Gỉam giá",
     "email": "E-mail",
+    "email.click_view" : "Xem ngay",
+    "email.message_order" : "Đơn hàng hôm nay chưa được duyệt là ",
+    "email.report_order" : "Số đơn hàng ch duyệt hôm nay",
```

sao ko làm 1 key 'email' riêng, 
```suggestion
    "email": {
        "email": "E-mail",
        "click_view" : "Xem ngay",
        "message_order" : "Đơn hàng hôm nay chưa được duyệt là ",
        "report_order" : "Số đơn hàng chưa duyệt hôm nay"
    }
```

---

dùng json nó k được dùng key nv ông a. trước test 1 lần rồi